chips/lowrisc/src/spi_host.rs

```diff
@@ -358,8 +358,11 @@ impl SpiHost {
 
     /// Enable both event/err IRQ
     fn enable_interrupts(&self) {
-        let regs = self.registers;
-        regs.intr_enable
+        self.registers
+            .intr_state
+            .write(intr::ERROR::SET + intr::SPI_EVENT::SET);
```

Comment:
Why do we need to set `INTR_STATE` here? Did you mean to `clear`?

Reply:
The [`spi_host.INTR_STATE`](https://docs.opentitan.org/hw/ip/spi_host/doc/#register-table) register is `rw1c` which means that writing 1 to it will clear the bit

Reply:
ah right!